glide.yaml

```diff
@@ -32,7 +32,6 @@ import:
   version: ca8abbf65d0e61dedf061f98bd3850f250e27539
 - package: github.com/samuel/go-thrift
   version: 2187045faa54fce7f5028706ffeb2f2fc342aa7e
-  # This is the same as labix.org/v2/mgo. See https://github.com/Masterminds/glide/issues/221#issuecomment-179817757
```

Comment:
Sounds like there isn't: https://github.com/Masterminds/glide/issues/691
I'll re-add it